src/dmd/dmodule.d

```diff
@@ -400,7 +400,7 @@ extern (C++) final class Module : Package
     Identifiers* versionidsNot; // forward referenced version identifiers
 
     Macro* macrotable;          // document comment macros
-    Escape* escapetable;        // document comment escapes
+    Escape escapetable;        // document comment escapes
```

Comment:
@Basile-z @thewilsonator Needs an update to https://github.com/dlang/dmd/blob/master/src/dmd/module.h as well.